network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/KeyPackageApiV0.kt

```diff
@@ -1,14 +1,19 @@
-package com.wire.kalium.network.api.keypackage
+package com.wire.kalium.network.api.v0.authenticated
 
 import com.wire.kalium.network.AuthenticatedNetworkClient
+import com.wire.kalium.network.api.base.authenticated.keypackage.ClaimedKeyPackageList
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackage
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackageApi
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackageCountDTO
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackageList
 import com.wire.kalium.network.utils.NetworkResponse
 import com.wire.kalium.network.utils.wrapKaliumResponse
 import io.ktor.client.request.get
 import io.ktor.client.request.parameter
 import io.ktor.client.request.post
 import io.ktor.client.request.setBody
 
-class KeyPackageApiImpl internal constructor(private val authenticatedNetworkClient: AuthenticatedNetworkClient) : KeyPackageApi {
+internal open class KeyPackageApiV0 internal constructor(private val authenticatedNetworkClient: AuthenticatedNetworkClient) : KeyPackageApi {
```

Comment:
We can later (after V2 introduced) move all MLS related APIs to V2, they shouldn't be accessible on V1 or V0.

Reply:
I will add a TODO for now and disable it when v2 is implemented